soda/core/soda/execution/check/distribution_check.py

```diff
@@ -115,31 +115,36 @@ def get_log_diagnostic_dict(self) -> dict:
         #     log_diagnostics.update(self.historic_diff_values)
         return log_diagnostics
 
-    def sql_column_values_query(self, distribution_check_cfg, limit=1000000) -> str:
+    def sql_column_values_query(self, distribution_check_cfg) -> str:
 
         column_name = distribution_check_cfg.column_name
-
         partition_filter = self.partition.sql_partition_filter
-        partition_str = ""
-
         distribution_check_filter = distribution_check_cfg.filter
+        sample_clause = distribution_check_cfg.sample_clause
+        scan = self.data_source_scan.scan
 
-        if distribution_check_filter:
-            scan = self.data_source_scan.scan
-            resolved_distribution_check_filter = scan.jinja_resolve(definition=distribution_check_filter)
-            partition_str = f"\nWHERE ({resolved_distribution_check_filter})"
-        if partition_filter:
-            scan = self.data_source_scan.scan
-            resolved_filter = scan.jinja_resolve(definition=partition_filter)
-            partition_str = (
-                f"{partition_str} AND ({resolved_filter})" if partition_str else f"\nWHERE {resolved_filter}"
+        filter_clause = None
+        limit = int(1e6)
+
+        if partition_filter and distribution_check_filter:
+            self.logs.error(
+                "Cannot have both dataset filter and distribution check filter at the same time",
```

Comment:
Why are we strict about this?

Reply:
For some reason, I was thinking that at soda-core it wasn't allowed to use both check filters and dataset filters at the same time. Indeed, it can be used together. I will fix it with `AND` operator

Reply:
Ok makes sense. At least for this check it works for sure since we're not doing crazy case and stuff. I think we don't even need to warn since people will know they have applied a filter from the dataset level due to the fact that they write:
`checks for dataset [filter_name]`

Reply:
done